CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Changelog
```

Comment:
Do we need to add 5.0.2 here?

Reply:
It appears here because this PR merges with the master branch from the `jenkinsci/
fortify-on-demand-uploader-plugin`
I don't think we should add it in this PR

Reply:
okay :)